classes/notification/NotificationAPIManager.php

```diff
@@ -163,6 +163,13 @@ public function getUserNotifications(): array
                         $notifications[] = NotificationDto::fromDb($row, $this->system->router->getUrl("inbox"));
                     }
                     break;
+                case "chat":
+                    if (false) {
```

Comment:
Is this supposed to be some other condition, or are we saving for later?

Reply:
Yeah, there's no default case for when a chat message should be deleted so keeping this a placeholder.

In the future we may decide to use the time created or set duration and delete if too old?